paimon-web-server/src/main/java/org/apache/paimon/web/server/data/result/enums/Status.java

```diff
@@ -60,10 +60,20 @@ public enum Status {
     /** ------------database-----------------. */
     DATABASE_NAME_IS_EXIST(10401, "database.name.exist"),
     DATABASE_CREATE_ERROR(10402, "database.create.error"),
+    DATABASE_REMOVE_ERROR(10403, "database.remove.error"),
 
     /** ------------table-----------------. */
     TABLE_NAME_IS_EXIST(10501, "table.name.exist"),
     TABLE_CREATE_ERROR(10502, "table.create.error"),
+    TABLE_ADD_COLUMN_ERROR(10503, "table.add.column.error"),
+    TABLE_ADD_OPTION_ERROR(10504, "table.add.option.error"),
+    TABLE_REMOVE_OPTION_ERROR(10505, "table.remove.option.error"),
+    TABLE_DROP_COLUMN_ERROR(10506, "table.drop.column.error"),
+    TABLE_AlTER_COLUMN_ERROR(10507, "table.alter.column.error"),
+    TABLE_UPDATE_COLUMN_TYPE_ERROR(10508, "table.update.column.type.error"),
+    TABLE_UPDATE_COLUMN_COMMENT_ERROR(10509, "table.update.column.comment.error"),
+    TABLE_DROP_ERROR(10510, "table.drop.error"),
+    TABLE_RENAME_ERROR(10510, "table.rename.error"),
```

Comment:
```suggestion
    TABLE_RENAME_ERROR(10510, "table.rename.error");
```
Meanwhile, removes the line 77.